tasks/eth/006-ecotone-set-gas-config/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+FOUNDATION_SAFE="0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A"
+SAFE_NONCE=""
```

Comment:
Let's hard code both this value and the one in the 005 ecotone ceremony.

Reply:
To what?

Reply:
```
`cast call 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A 'nonce()(uint)'`
90
```

So, this one should be 91.

For simplicity we can leave 005 asis.

Reply:
How do we simulate this when we set it to 91? IIUC CI would fail with this.

Reply:
CI should not fail. 
1. if a custom SAFE_NONCE is specified, it will be added to the State Overrides used in the simulation
2. CI just tests that the `just simulate` invocation passes, and unfortunately does not have the ability to simulate the actual execution result. #34 or something similar will fix this some day.

Reply:
got it, set it to 91